src/test/java/com/sap/cloudfoundry/client/facade/ServicesCloudControllerClientIntegrationTest.java

```diff
@@ -226,7 +227,7 @@ private void pollLastOperationServiceInstanceState(String serviceInstanceName) t
                                                .getLastOperation();
         while (!lastOperation.getState()
                              .equals(ServiceOperation.State.SUCCEEDED)) {
-            Thread.sleep(TimeUnit.SECONDS.toMillis(1));
+            sleep(1);
```

Comment:
Replace value 1 with timeunit constant. Sleep with value "1" is not obvious what time unit is used.